servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2ToStH1ClientDuplexHandler.java

```diff
@@ -65,6 +65,7 @@
     @Override
     public void write(ChannelHandlerContext ctx, Object msg, ChannelPromise promise) {
         if (msg instanceof HttpRequestMetaData) {
+            closeHandler.protocolPayloadBeginOutbound(ctx);
```

Comment:
`HttpObjectEncoder` emits this event right before `ctx.write(serializedData)`. Consider making them consistent. We can move it in `HttpObjectEncoder` to the top, before `T metaData = castMetaData(msg);`

Reply:
I'll move it after the case bcz the `closeHandler.protocolClosingOutbound(ctx);` needs the `metadata`